tekton/publish-nightly.yaml

```diff
@@ -105,7 +105,7 @@ spec:
       DATE=$(date +"%Y%m%d")
       VERSION_TAG="$DATE-$COMMIT"
 
-      echo $VERSION_TAG > "/builder/home/version"
+      echo $VERSION_TAG > "/tekton/home/version"
```

Comment:
I just realized that since this runs on Tekton 0.3.1, there will be no /tekton/home there :(